src/Theme.tsx

```diff
@@ -31,6 +31,11 @@ const breakpoints = {
 export const themeProps = {
   borders: [0, "2px solid"],
 
+  /**
+   *  This allows styled-system to hook into our breakpoints
+   */
+  breakpoints: [breakpoints.sm, breakpoints.md, breakpoints.lg, breakpoints.xl],
```

Comment:
👍

Reply:
This is missing xs... was that intentional?

Reply:
Neeeever mind. I see what it's doing. `xs` is just whatever is smaller than `sm`. 👍